tests/config.tests/applied.json

```diff
@@ -3,17 +3,17 @@
   "blobset": {
     "blobs": {
       "arbitrary_manufacturer_id": {
```

Comment:
Believe I deleted this file and moved the test to more specific filename blobset...

Reply:
They are still in the master branch of faucetsdn -- https://github.com/faucetsdn/udmi/blob/master/tests/config.tests/applied.json

I can delete them, but they haven't been deleted as of yet.